openlibrary/catalog/marc/parse.py

```diff
@@ -2,7 +2,9 @@
 
 from openlibrary.catalog.marc.get_subjects import subjects_for_work
 from openlibrary.catalog.marc.marc_base import BadMARC, NoTitle, MarcException
-from openlibrary.catalog.utils import pick_first_date, tidy_isbn, flip_name, remove_trailing_dot, remove_trailing_number_dot
```

Comment:
this seems out of place for this PR, but it looks like nothing is deleted, just moved to multiple lines)

Reply:
@mekarpeles I thought I will get this error E501 line too long (113 > 88 characters) that,s why I moved it to multiple lines.